packages/SwingSet/src/controller/startXSnap.js

```diff
@@ -53,13 +56,15 @@ export function makeStartXSnap(bundles, options) {
   /**
    * @param {string} vatID
    * @param {string} name
+   * @param {import('../types-external.js').BundleID[]} bundleIDs
    * @param {(request: Uint8Array) => Promise<Uint8Array>} handleCommand
    * @param {boolean} [metered]
    * @param {boolean} [reload]
    */
   async function startXSnap(
     vatID,
     name,
+    bundleIDs,
     handleCommand,
     metered,
     reload = false,
```

Comment:
This one might be a job for another day, but just to register it:
```suggestion
  /**
   * @param {string} vatID
   * @param {string} name
   * @param {object} details
   * @param {import('../types-external.js').BundleID[]} details.bundleIDs
   * @param {(request: Uint8Array) => Promise<Uint8Array>} details.handleCommand
   * @param {boolean} [details.metered]
   * @param {boolean} [details.reload]
   */
  async function startXSnap(vatID, name, {
    bundleIDs,
    handleCommand,
    metered,
    reload = false,
```